### PR TITLE
Sign-in/-Sign-up-Page

### PR DESCRIPTION
**Pull request by khaled106**
Created SignIn/SignUp Page
There is still a bug to be fixed in the css ("A new issue wil be created for this") 
Now the data will be stored in the console.

### CODE REVIEW

File: client/src/components/SignIn/SignIn.css [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+/* border-radius: 12px; */
+/* border: 0; */
+/* box-sizing: border-box; */
+/* color: #eee; */
+/* font-size: 18px; */
+/* height: 100%; */
+/* outline: 0; */
+/* padding: 4px 20px 0; */
+/* width: 100%; */
+/* } */
```
Comment (ramy-att) [inline client/src/components/SignIn/SignIn.css:65]: @khaled106 can you please remove all commented out code

File: client/src/components/SignUp/SignUp.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
+/* body { */
+/* align-items: center; */
+/* background-color: #000; */
+/* display: flex; */
+/* justify-content: center; */
+/* height: 100vh; */
+/* } */
+/*    */
```
Comment (ramy-att) [inline client/src/components/SignUp/SignUp.css:8]: We can remove extra comments

**Review by ramy-att — CHANGES_REQUESTED** [review]
There is a lot of commented-out code we can just remove that. The functionality looks great! I will create an issue for the small bug on the label's transition and spacing

File: client/src/components/SignUp/SignUp.css (hunk 2 above)
Comment (khaled106) [reply client/src/components/SignUp/SignUp.css:8]: Done

**Comment by khaled106** [comment]
@ramy-att  I removed the unnecessary comments, you can check over the code again and merge to master if it's good.
Thanks Ramy !!

**Review by ramy-att — APPROVED** [review]
Looks amazing.

**Comment by ramy-att** [comment]
> @ramy-att I removed the unnecessary comments, you can check over the code again and merge to master if it's good. Thanks Ramy !!

Looks good

**Pull request merged**